kubemarine/haproxy.py

```diff
@@ -68,11 +82,24 @@ def enrich_inventory(inventory, cluster):
     return inventory
 
 
+def get_config_path(group: NodeGroup) -> NodeGroupResult:
+    with RemoteExecutor(group.cluster) as exe:
+        for node in group.get_ordered_members_list(provide_node_configs=True):
+            package_associations = get_associations_for_node(node)
+            cmd = f"systemctl show -p MainPID {package_associations['service_name']} " \
+                  f"| cut -d '=' -f2 " \
+                  f"| xargs -I PID sudo cat /proc/PID/environ " \
+                  f"| tr '\\0' '\\n' | grep CONFIG | cut -d \"=\" -f2 | tr -d '\\n'"
```

Comment:
Use the same way to detect currently used config as external-traffic-manager does.